python/cudf/cudf/_lib/pylibcudf/types.pyx

```diff
@@ -67,3 +68,7 @@ cdef class DataType:
         cdef DataType ret = DataType.__new__(DataType, type_id.EMPTY)
         ret.c_obj = dt
         return ret
+
+
+SIZE_TYPE = DataType(type_to_id[size_type]())
```

Comment:
This seems like something that could probably be used much more broadly than it is in this PR. Not in scope, but something we might want to come back to and clean up.

Reply:
Yeah, I was just introducing it here really.